lib/address_geocoder.rb

```diff
@@ -56,7 +56,7 @@ def call_google
         if attempts <= 5
           retry
         else
-          raise ConnectionError, 'Could not connect to GoogleAPI'
+          raise SystemCallError, 'Could not connect to GoogleAPI'
```

Comment:
Oh, well guess that doesn't work. Need to pick some other one or create our own then...Maybe ThreadError or TypeError?
<img width="287" alt="screen shot 2016-01-24 at 10 57 04 pm" src="https://cloud.githubusercontent.com/assets/12767276/12536924/d3bd02d4-c2ed-11e5-889a-fa2b9fc45a3b.png">